diff --git a/node_modules/@angular/platform-browser/fesm2022/platform-browser.mjs b/node_modules/@angular/platform-browser/fesm2022/platform-browser.mjs
index 32a7f41..3c0cb6c 100755
--- a/node_modules/@angular/platform-browser/fesm2022/platform-browser.mjs
+++ b/node_modules/@angular/platform-browser/fesm2022/platform-browser.mjs
@@ -629,14 +629,18 @@ i0.ɵɵngDeclareClassMetadata({
   }]
 });
 
-var HydrationFeatureKind;
-(function (HydrationFeatureKind) {
-  HydrationFeatureKind[HydrationFeatureKind["NoHttpTransferCache"] = 0] = "NoHttpTransferCache";
-  HydrationFeatureKind[HydrationFeatureKind["HttpTransferCacheOptions"] = 1] = "HttpTransferCacheOptions";
-  HydrationFeatureKind[HydrationFeatureKind["I18nSupport"] = 2] = "I18nSupport";
-  HydrationFeatureKind[HydrationFeatureKind["EventReplay"] = 3] = "EventReplay";
-  HydrationFeatureKind[HydrationFeatureKind["IncrementalHydration"] = 4] = "IncrementalHydration";
-})(HydrationFeatureKind || (HydrationFeatureKind = {}));
+const HydrationFeatureKind = {
+  NoHttpTransferCache: 0,
+  HttpTransferCacheOptions: 1,
+  I18nSupport: 2,
+  EventReplay: 3,
+  IncrementalHydration: 4
+};
+HydrationFeatureKind[HydrationFeatureKind.NoHttpTransferCache] = 'NoHttpTransferCache';
+HydrationFeatureKind[HydrationFeatureKind.HttpTransferCacheOptions] = 'HttpTransferCacheOptions';
+HydrationFeatureKind[HydrationFeatureKind.I18nSupport] = 'I18nSupport';
+HydrationFeatureKind[HydrationFeatureKind.EventReplay] = 'EventReplay';
+HydrationFeatureKind[HydrationFeatureKind.IncrementalHydration] = 'IncrementalHydration';
 function hydrationFeature(ɵkind, ɵproviders = [], ɵoptions = {}) {
   return {
     ɵkind,
diff --git a/node_modules/@angular/platform-browser/types/_browser-chunk.d.ts b/node_modules/@angular/platform-browser/types/_browser-chunk.d.ts
index fe2126a..d82e7b8 100755
--- a/node_modules/@angular/platform-browser/types/_browser-chunk.d.ts
+++ b/node_modules/@angular/platform-browser/types/_browser-chunk.d.ts
@@ -5,7 +5,7 @@
  */
 
 import * as i0 from '@angular/core';
-import { PlatformRef, Type, ApplicationConfig, ApplicationRef, Provider, StaticProvider } from '@angular/core';
+import { PlatformRef, Type, ApplicationConfig, ApplicationRef, Provider, StaticProvider, Injector } from '@angular/core';
 import * as i1 from '@angular/common';
 
 /**
@@ -114,7 +114,7 @@ declare function provideProtractorTestingSupport(): Provider[];
  *
  * @publicApi
  */
-declare const platformBrowser: (extraProviders?: StaticProvider[]) => PlatformRef;
+declare const platformBrowser: (extraProviders?: StaticProvider[], parentInjector?: Injector) => PlatformRef;
 /**
  * Exports required infrastructure for all Angular apps.
  * Included by default in all Angular apps created with the CLI
diff --git a/node_modules/@angular/platform-browser/types/testing.d.ts b/node_modules/@angular/platform-browser/types/testing.d.ts
index c12e21b..1802c60 100755
--- a/node_modules/@angular/platform-browser/types/testing.d.ts
+++ b/node_modules/@angular/platform-browser/types/testing.d.ts
@@ -5,7 +5,7 @@
  */
 
 import * as i0 from '@angular/core';
-import { StaticProvider, PlatformRef } from '@angular/core';
+import { StaticProvider, PlatformRef, Injector } from '@angular/core';
 import { BrowserModule } from './_browser-chunk.js';
 import '@angular/common';
 
@@ -14,7 +14,7 @@ import '@angular/common';
  *
  * @publicApi
  */
-declare const platformBrowserTesting: (extraProviders?: StaticProvider[]) => PlatformRef;
+declare const platformBrowserTesting: (extraProviders?: StaticProvider[], parentInjector?: Injector) => PlatformRef;
 /**
  * NgModule for testing.
  *
