diff --git a/node_modules/@angular/platform-browser-dynamic/platform-browser-dynamic.d.ts b/node_modules/@angular/platform-browser-dynamic/platform-browser-dynamic.d.ts
index 454600d..c954a67 100755
--- a/node_modules/@angular/platform-browser-dynamic/platform-browser-dynamic.d.ts
+++ b/node_modules/@angular/platform-browser-dynamic/platform-browser-dynamic.d.ts
@@ -45,7 +45,7 @@ export declare class JitCompilerFactory implements CompilerFactory {
 /**
  * @publicApi
  */
-export declare const platformBrowserDynamic: (extraProviders?: StaticProvider[] | undefined) => PlatformRef;
+export declare const platformBrowserDynamic: (extraProviders?: StaticProvider[] | undefined, parentInjector?: Injector) => PlatformRef;
 
 /**
  * @publicApi
diff --git a/node_modules/@angular/platform-browser-dynamic/testing/testing.d.ts b/node_modules/@angular/platform-browser-dynamic/testing/testing.d.ts
index 8f20631..7750cb2 100755
--- a/node_modules/@angular/platform-browser-dynamic/testing/testing.d.ts
+++ b/node_modules/@angular/platform-browser-dynamic/testing/testing.d.ts
@@ -24,7 +24,7 @@ export declare class BrowserDynamicTestingModule {
 /**
  * @publicApi
  */
-export declare const platformBrowserDynamicTesting: (extraProviders?: StaticProvider[] | undefined) => PlatformRef;
+export declare const platformBrowserDynamicTesting: (extraProviders?: StaticProvider[] | undefined, parentInjector?: Injector) => PlatformRef;
 
 /**
  * A DOM based implementation of the TestComponentRenderer.
