diff --git a/node_modules/@angular/platform-browser-dynamic/index.d.ts b/node_modules/@angular/platform-browser-dynamic/index.d.ts
index f772e20..e703efa 100755
--- a/node_modules/@angular/platform-browser-dynamic/index.d.ts
+++ b/node_modules/@angular/platform-browser-dynamic/index.d.ts
@@ -8,6 +8,7 @@
 import { Compiler } from '@angular/core';
 import { CompilerFactory } from '@angular/core';
 import { CompilerOptions } from '@angular/core';
+import { Injector } from '@angular/core';
 import { PlatformRef } from '@angular/core';
 import { Provider } from '@angular/core';
 import { StaticProvider } from '@angular/core';
@@ -29,7 +30,7 @@ export declare class JitCompilerFactory implements CompilerFactory {
 /**
  * @publicApi
  */
-export declare const platformBrowserDynamic: (extraProviders?: StaticProvider[] | undefined) => PlatformRef;
+export declare const platformBrowserDynamic: (extraProviders?: StaticProvider[] | undefined, parentInjector?: Injector) => PlatformRef;
 
 /**
  * @publicApi
diff --git a/node_modules/@angular/platform-browser-dynamic/testing/index.d.ts b/node_modules/@angular/platform-browser-dynamic/testing/index.d.ts
index 7f57eed..53ba89f 100755
--- a/node_modules/@angular/platform-browser-dynamic/testing/index.d.ts
+++ b/node_modules/@angular/platform-browser-dynamic/testing/index.d.ts
@@ -7,6 +7,7 @@
 
 import * as i0 from '@angular/core';
 import * as i1 from '@angular/platform-browser/testing';
+import { Injector } from '@angular/core';
 import { PlatformRef } from '@angular/core';
 import { StaticProvider } from '@angular/core';
 import { TestComponentRenderer } from '@angular/core/testing';
@@ -25,7 +26,7 @@ export declare class BrowserDynamicTestingModule {
 /**
  * @publicApi
  */
-export declare const platformBrowserDynamicTesting: (extraProviders?: StaticProvider[] | undefined) => PlatformRef;
+export declare const platformBrowserDynamicTesting: (extraProviders?: StaticProvider[] | undefined, parentInjector?: Injector) => PlatformRef;
 
 /**
  * A DOM based implementation of the TestComponentRenderer.
