diff --git a/node_modules/@angular/platform-browser/platform-browser.d.ts b/node_modules/@angular/platform-browser/platform-browser.d.ts
index f18782e..9c9738b 100755
--- a/node_modules/@angular/platform-browser/platform-browser.d.ts
+++ b/node_modules/@angular/platform-browser/platform-browser.d.ts
@@ -520,7 +520,7 @@ export declare type MetaDefinition = {
  *
  * @publicApi
  */
-export declare const platformBrowser: (extraProviders?: StaticProvider[]) => PlatformRef;
+export declare const platformBrowser: (extraProviders?: StaticProvider[], parentInjector?: Injector) => PlatformRef;
 
 /**
  * Marker interface for a value that's safe to use as HTML.
