diff --git a/node_modules/ngx-monaco-editor/fesm2015/ngx-monaco-editor.js b/node_modules/ngx-monaco-editor/fesm2015/ngx-monaco-editor.js
index 3ebd1bf..1ff11ed 100644
--- a/node_modules/ngx-monaco-editor/fesm2015/ngx-monaco-editor.js
+++ b/node_modules/ngx-monaco-editor/fesm2015/ngx-monaco-editor.js
@@ -293,6 +293,7 @@ EditorComponent.decorators = [
     { type: Component, args: [{
                 selector: 'ngx-monaco-editor',
                 template: '<div class="editor-container" #editorContainer></div>',
+                standalone: false,
                 providers: [{
                         provide: NG_VALUE_ACCESSOR,
                         useExisting: forwardRef((/**
@@ -435,6 +436,7 @@ DiffEditorComponent.decorators = [
     { type: Component, args: [{
                 selector: 'ngx-monaco-diff-editor',
                 template: '<div class="editor-container" #editorContainer></div>',
+                standalone: false,
                 styles: [`
     :host {
       display: block;
